libpod/image_inspect.go

```diff
@@ -4,18 +4,22 @@ import (
 	"encoding/json"
 	"strings"
 
-	"github.com/containers/image/types"
-	"github.com/containers/storage"
 	digest "github.com/opencontainers/go-digest"
 	ociv1 "github.com/opencontainers/image-spec/specs-go/v1"
 	"github.com/pkg/errors"
+	"github.com/projectatomic/libpod/libpod/image"
 	"github.com/projectatomic/libpod/pkg/inspect"
 )
 
-func getImageData(img storage.Image, imgRef types.Image, size int64, driver *inspect.Data) (*inspect.ImageData, error) {
-	imgSize, err := imgRef.Size()
+// GetImageData returns an image's inspect data
+func GetImageData(img *image.Image) (*inspect.ImageData, error) {
```

Comment:
Already noted on IRC, but I'd love to see this moved over to the image package at some point

Reply:
Also, how is this different from `Inspect`? It seems to do roughly the same thing?

Reply:
Alright, so this is actually used for inspecting images. Can we rename it to `Inspect` then?

Reply:
yeah long term it will be as such ... leave for now. next PR hits it